sihl_users/lib/command.ml

```diff
@@ -8,8 +8,7 @@ let fn request args =
   | [ "createadmin"; email; password ] -> (
       let* result =
         try_to_run (fun () ->
-            Service.User.create_admin request ~email ~password ~username:"admin"
-              ~name:"admin")
+            Service.User.create_admin request ~email ~password ~username:None)
```

Comment:
@jerben Why set this to `None`?

Reply:
Why make it optional?

Reply:
@aronerben Most of the services don't force me to have a username, but email seems to be mandatory pretty much everywhere. Also we can later on use OAuth to pull information from Github, Google, Facebook etc. while most of the time we won't get a username.

Reply:
Ok, makes sense, ty!